packages/project-disaster-trail/src/state/user.js

```diff
@@ -26,9 +26,11 @@ export const addPoints = points => dispatch => {
 // REDUCERS
 export const user = createReducer(initialState, {
   [actionTypes.SET_POINTS]: (state, action) => {
+    /* eslint-disable no-param-reassign */
     state.points = action.points;
   },
   [actionTypes.ADD_POINTS]: (state, action) => {
+    /* eslint-disable no-param-reassign */
```

Comment:
I've found it easiest to disable `no-param-reassign` for entire reducer blocks since it will happen for every action

Reply:
Good idea!